README.md

```diff
@@ -117,10 +119,24 @@ How do you move from `labs` back up to your `code` directory? You can always mov
 
 `labs` is within `code`. From within `labs`, you would refer to the parent directory, `code` as `..`.
 
-In the same manner that `..` means the directory above, the shortcut `.` means the current directory. You'll see why being able to refer to your current directory as `.` is helpful in a minute. 
+In the same manner that `..` means the directory above, the shortcut `.` means the current directory. You'll see why being able to refer to your current directory as `.` is helpful in a minute.
 
 ### Hint: Tab Autocomplete
 
 When you're in Terminal, to autocomplete a directory or a command, start typing and then press TAB.
 
+## Key Terms
```

Comment:
I think this might be better as a Glossary: just short blurbs. Or "Cheat Sheet?"

Reply:
Meh. Not that convinced Key Terms isn't better on second thought. Your choice.